gateways/stripe/stripe_vars.php

```diff
@@ -134,29 +141,29 @@ function espresso_display_stripe($payment_data) {
  */
 function espresso_get_stripe_decimal_places($currency = '')
 {
-    if (!$currency) {
-    	$stripe_settings = get_option('event_espresso_stripe_settings');
-        $currency = !empty($stripe_settings['stripe_currency_symbol']) ? $stripe_settings['stripe_currency_symbol'] : 'USD';
-    }
-    switch (strtoupper($currency)) {
-        // Zero decimal currencies.
-        case 'BIF' :
-        case 'CLP' :
-        case 'DJF' :
-        case 'GNF' :
-        case 'JPY' :
-        case 'KMF' :
-        case 'KRW' :
-        case 'MGA' :
-        case 'PYG' :
-        case 'RWF' :
-        case 'VND' :
-        case 'VUV' :
-        case 'XAF' :
-        case 'XOF' :
-        case 'XPF' :
-            return 0;
-        default :
-            return 2;
-    }
+	if (!$currency) {
```

Comment:
hmmm this is new though. At least for EE4 we're using spaces instead of tabs (see https://github.com/eventespresso/event-espresso-core/blob/master/docs/A--Best-Practices/php-standards.md#white-space and https://www.php-fig.org/psr/psr-2/).

Reply:
Thanks, Mike. I just went with tabs for consistency.

Switching between both in the same file was annoying me so I converted everything.